Clients/src/presentation/components/Table/index.tsx

```
@@ -13,32 +13,47 @@ import {
 import { getAllVendors } from "../../../application/repository/vendor.repository";
 
 const DEFAULT_ROWS_PER_PAGE = 10;
 const RISKS_ROWS_PER_PAGE_KEY = 'verifywise_risks_rows_per_page';
 
+// Define proper interfaces for type safety
+interface TableColumn {
+  id: string;
+  name: string;
+}
+
+interface TableRow {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TableRow'. Consider to delete it or rename it.

'TableRow' is defined here:


(lint/suspicious/noRedeclare)